modules/broker-auth/src/main/java/io/ballerina/messaging/broker/auth/BrokerAuthConstants.java

```diff
@@ -54,4 +54,7 @@ private BrokerAuthConstants() {
     public static final String USERS_CONFIG_NAMESPACE = "wso2.broker.users";
     // user manager property name
     public static final String PROPERTY_USER_STORE_CONNECTOR = "broker.user.store.connector";
+    public static final String USER_STORE_MANAGER_PROPERTY = "UserStoreManagerImpl";
```

Comment:
Can we remove this constant? It is not used anywhere in the repo.

Reply:
Fixed in 384b733